readme.md

```diff
@@ -367,6 +368,41 @@ AVA comes with builtin support for ES2015 through [Babel 6](https://babeljs.io).
 
 AVA includes typings for TypeScript. You have to setup transpilation yourself. When you set `module` to `commonjs` in your `tsconfig.json` file, TypeScript will automatically find the type definitions for AVA. You should set `target` to `es2015` to use Promises and async functions.
 
+If you want to customize the babel transpiler for test files, you can do so by adding a `"babel"` key to the `"ava"` section in your package.json file.
```

Comment:
Some nits:

* This needs a heading.
* The README isn't entirely consistent but I'd prefer `package.json` (with the backticks). Same for `.babelrc`.
* It's named "`ava` section" (without quotes) earlier: https://github.com/spudly/ava/blob/f928959b9ad106a70fc3091a462ff3d96a60f469/readme.md#configuration
* Could you capitalize Babel throughout?